### PR TITLE
Added TLDRPages: simplified man pages

### PR DESCRIPTION
**Pull request by mattr555**
[TLDR Pages](https://github.com/tldr-pages/tldr) are simplified man pages and are a good candidate for a fathead!

I'm not sure how this will turn out, as I can't test fatheads.

https://duck.co/ia/view/tldr_pages

### CODE REVIEW

**Comment by jagtalon** [comment]
@mattr555 I'm excited about this!

**Comment by jdorweiler** [comment]
@mattr555 sorry I still haven't automated fatheads to install on beta yet.  I'll work on getting it up there though.

File: share/fathead/tldr_pages/output.txt [hunk 1]
```diff
@@ -0,0 +1,193 @@
+7za (A file archiver with high compression ratio)	A										compress directory or file\n<pre><code>7za a {{compressed.7z}} {{directory_or_file_to_compress}}</code></pre>\ndecompress an existing 7z file with original directory structure\n<pre><code>7za x {{compressed.7z}}</code></pre>\ncompress to zip format\n<pre><code>7za a -tzip {{compressed.zip}} {{directory_or_file_to_compress}}</code></pre>\ncreate multipart 7zip file; `part_size` specifies part size in Bytes, Kilobytes, Megabytes or Gigabytes.\n<pre><code>7za -v{{part_size}}{{[b|k|m|g]}} {{compressed.7z}} {{directory_or_file_to_compress}}</code></pre>	https://github.com/tldr-pages/tldr/blob/master/pages/common/7za.md
```
Comment (jdorweiler) [inline share/fathead/tldr_pages/output.txt]: Can you remove the stuff in the parens.  The first field is the search query so someone would have to search `7za (A file archiver with high compression ratio)` for this to work.

**Comment by mattr555** [comment]
@jdorweiler done. My goal was to have that parenthetical displayed as the subtitle but I decided that it might just be better off without it altogether.

**Comment by jdorweiler** [comment]
@mattr555 Oh ok. I forget how subtitles work :(  Did you see anything in the docs or another fathead about them?

**Comment by moollaza** [comment]
@jdorweiler Maybe @mrshu knows?

**Comment by mrshu** [comment]
Hello everyone, let's see if I can help you out here.

Unfortunately, I believe that subtitles currently come from the metadata and are set on per fathead basis, such as in case of [emacs](https://duckduckgo.com/?q=emacs+quoted+insert&ia=about), [Can I Use](https://duckduckgo.com/?q=caniuse+indexeddb&ia=about&iax=1) or [Apache](https://duckduckgo.com/?q=apache+acceptmutex&ia=about).

On the other hand, while the first field is indeed the search query one of the subsequent fields (actually the second one in the final output that gets inserted into the DB) holds the "human readable" title so if you wanted to include the text in parentheses this might be a good place for it. Whatever is in the parenthesis will become the subtitle.

I am sure @jdorweiler can help you out here as I do not know the DuckDuckHack fathead process that well.

Hope this helps!

**Comment by jdorweiler** [comment]
@mattr555 I installed it on beta. 
https://beta.duckduckgo.com/?q=tldr+tar&ia=about

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=tldr tar)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/tldr_pages_mobile.png)

**Comment by mattr555** [comment]
Looks pretty good! To be honest I think it does its job without the subtitle :+1:

**Comment by jdorweiler** [comment]
:+1:  agreed.  Thanks @mattr555 
I'll let you know when it goes live.

**Comment by jdorweiler** [comment]
@mattr555 It's live :cake: 
https://duckduckgo.com/?q=tldr+chmod&t=canonical&ia=about

**Comment by jdorweiler** [comment]
I added example(s) as a trigger word too 
https://duckduckgo.com/?q=chmod+example&t=canonical&ia=about&iax=1

**Pull request merged**